includes/class-indieauth-admin.php

```diff
@@ -170,6 +170,16 @@ public function settings() {
 				'default'      => 0,
 			)
 		);
+		register_setting(
+			'indieauth',
+			'indieauth_root_user',
+			array(
+				'type'         => 'int',
+				'description'  => __( 'User Who is Represented by the Site URL', 'indieauth' ),
+				'show_in_rest' => true,
+				'default'      => get_option( 'iw_default_author', 0 ),
```

Comment:
what if there is no user with the id `0`?

Reply:
Seems to be a similar discussion we had in the #indieweb-wordpress chat.

Reply:
or is it showing `None` in this case?

Reply:
can we use the same mechanism in the indieweb plugin?

Reply:
It's showing none. Then it goes back to using author urls or checking for a single user.